docs/README.md

```diff
@@ -13,5 +13,5 @@ Available storage and pricing is not controlled by any single company. Instead,
 
 There are many ways to participate in the Filecoin network, and several libraries, tools, products, and services to choose from:
 
-- If you're new to web3 and Filecoin, we recommend checking out the resources in [this guide](/introduction/new-to-web3.md).
+- If you're new to web3 and Filecoin, we recommend checking out [What is Filecoin?](/introduction/what-is-filecoin.md).
```

Comment:
```suggestion
- If you're new to web3 and Filecoin, we recommend checking out [What is Filecoin?](./introduction/what-is-filecoin).
```

Relative links keep @cwaring happy.

Reply:
...they also need to have the filename extension too, otherwise they don't work `./introduction/what-is-filecoin.md` 😎